src/h5web/visualizations/heatmap/HeatmapVis.tsx

```diff
@@ -21,8 +21,20 @@ function HeatmapVis(props: Props): JSX.Element {
   const { rows, cols } = getDims(dataArray);
   const values = dataArray.data as number[];
 
-  const [keepAspectRatio, showGrid] = useHeatmapConfig(
-    (state) => [state.keepAspectRatio, state.showGrid],
+  const [
+    dataDomain,
+    customDomain,
+    scaleType,
+    keepAspectRatio,
+    showGrid,
+  ] = useHeatmapConfig(
+    (state) => [
+      state.dataDomain,
+      state.customDomain,
+      state.scaleType,
+      state.keepAspectRatio,
+      state.showGrid,
+    ],
```

Comment:
Isn't this basically the whole config now? `const { ... } = useHeatmapConfig()` would be a lot cleaner, I reckon.

Reply:
Well, there are still the setters inside the config, but we might as well cross the bridge

Reply:
They can't change so they can't trigger re-renders.